.pre-commit-config.yaml

```diff
@@ -48,3 +48,8 @@ repos:
     hooks:
     -   id: mypy
         exclude: tests/
+-   repo: https://github.com/PrincetonUniversity/blocklint
+    rev: v0.2.4
+    hooks:
+    -   id: blocklint
+        exclude: 'docs/Makefile|docs/release_notes.rst|tox.ini'
```

Comment:
Does this find all other files in the repo? I think I was trying with listing directories to include but if this works I like it better.

Reply:
I've tried sticking blocked words in various files and it hasn't missed one yet.